vars/defaults.yml

```diff
@@ -5,6 +5,7 @@
 ###############################################################################
 
 # # common
+# common_timezone: 'Etc/UTC' (required)
```

Comment:
Why mark it (required) then? I think most people would be 

Sorry, I wasn't clear. It's including the line in user.yml that seems
unnecessary -- it implies users need to override it in general. Why not
drop the line from user.yml (i.e., have it be a variable where the
default is fine, as is the case for ``main_user_shell: "/bin/bash"``, which is only
in defaults and not in user.yml)


On 11/06, Laurent Arnoud wrote:
> > @@ -5,6 +5,7 @@
> >  ###############################################################################
> >  
> >  # # common
> > +# common_timezone: 'Etc/UTC' (required)
> 
> No don't think so its already currently the default. I will set it as default then.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/sovereign/sovereign/pull/424/files#r44146509